### PR TITLE
feat(bug_fixes): Update readme with new repo information

### PR DESCRIPTION
**Pull request by Thexumaker**
- Updates broken links
- Updates wiki
-  Moves to github actions
- update tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,10 +131,12 @@ Since Lob charges per request, you can set `delaySearch` to true which delays an
 | Introduction of Autocomplete Forms |
 | Removed UI restrictions blocking flow of autocomplete results | 
 ### 1.0.7
+| 1.0.7 |
```
Comment (mmrtnz) [inline README.md:134]: Our package.json updates to 1.0.10, did we goof up somewhere and forget to document release logs for 1.0.8 and 1.0.9?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+name: Workflow Test
```
Comment (mmrtnz) [inline .github/workflows/test.yml:1]: Noiiiiice 💯

File: src/Autocomplete.test.js [hunk 3]
```diff
@@ -86,39 +84,6 @@ describe('Autocomplete', () => {
...
-          primary_line: "123 Micky's Clubhouse",
-          state: 'FL',
-          zip_code: '12345'
-        }
-      }
-    ]
-
-    expect(handleSuggestion).toHaveBeenCalled()
-    expect(handleSuggestion).toHaveBeenCalledWith(expectedSuggestions)
-
```
Comment (mmrtnz) [inline src/Autocomplete.test.js:121]: What's the reason for removing this test scenario? If it's a temporary bug let's comment this out instead

**Review by mmrtnz — APPROVED** [review]
Looks good! Just a couple questions

File: src/Autocomplete.test.js (hunk 3 above)
Comment (Thexumaker) [reply src/Autocomplete.test.js:121]: We removed the onSuggestion prop in the react component

File: README.md (hunk 1 above)
Comment (Thexumaker) [reply README.md:134]: yeah I'm partially to blame for that one lol, ran npm publish a couple times extra

**Pull request merged**